ldes-server-domain/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/snapshot/services/SnapshotServiceImplTest.java

```diff
@@ -36,16 +36,15 @@ class SnapshotServiceImplTest {
 	private final SnapshotRepository snapshotRepository = mock(SnapshotRepository.class);
 	private final LdesFragmentRepository ldesFragmentRepository = mock(LdesFragmentRepository.class);
 	private final SnapshotRelationLinker snapshotRelationLinker = mock(SnapshotRelationLinker.class);
-	private final ViewService viewService = mock(ViewService.class);
+	private final EventStreamService eventStreamService = mock(EventStreamService.class);
```

Comment:
can you add a unit test case where you verify tthe flow that throws an exception? (You will have to change the flow first based on my remark above)